volatility3/framework/plugins/windows/threads.py

```diff
@@ -36,9 +36,11 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             ),
         ]
 
-    @classmethod
+    @staticmethod
```

Comment:
Nope, staticmethods are bad.   It means you then can't get back to the calling class (for instance, to find version information if you need to).  So this is a red flags.